### ISSUE TITLE
Contrast for flash messages too low

### ISSUE THREAD

**Issue by ClearlyClaire** [mastodon/mastodon#9410]
The contrast of “flash messages” in forms is too low. Furthermore, since the background of the flash message box uses transparent background, it's hard to account for when designing themes.

## Default theme

![screenshot_2018-12-02 mastodon instance perso de thibg](https://user-images.githubusercontent.com/384364/49339188-6cdab080-f62e-11e8-9411-bdc6eeb917fa.png)

https://webaim.org/resources/contrastchecker/?fcolor=DF405A&bcolor=56313F

## Light theme

![screenshot_2018-12-02 mastodon instance perso de thibg 1](https://user-images.githubusercontent.com/384364/49339189-73692800-f62e-11e8-91f0-9413a89e9f85.png)

https://webaim.org/resources/contrastchecker/?fcolor=DF405A&bcolor=DBB9C4

**Comment by ThibG** [comment]
Reopening since while #13892 is a significant improvement, the contrast is still somewhat low.

**Comment by cchoi12** [comment]
I played around with the tool and used some references elsewhere, 
I tried out https://webaim.org/resources/contrastchecker/?fcolor=000000&bcolor=CF6679 which would pass the large text but not the small(all font-weight in the flash messages are bold anyways). 

Looks like this:
![Screen Shot 2020-05-31 at 2 05 14 PM](https://user-images.githubusercontent.com/19676120/83364544-d9835800-a356-11ea-91cc-0af15bafc2af.png)

But again, this deviates away from the original color palette which may make it too drastic of a change. Just throwing it up there anyways.
